civis/tests/test_io.py

```diff
@@ -419,3 +433,24 @@ def test_split_schema_tablename_raises():
     s = "table.with.too.many.periods"
     with pytest.raises(ValueError):
         civis.io._tables.split_schema_tablename(s)
+
+
+@mock.patch.object(civis.io._tables, '_sql_script', autospec=True,
+                   return_value=[-1000, 1000])
+@mock.patch.object(civis.io._tables, 'APIClient', autospec=True)
+def test_export_to_civis_file(mock_APIClient, mock_sql_script):
+    expected = [{'file_id': 9844453}]
+
+    # Mock client.scripts.get_sql_runs to return successful response with data
+    # (mock_client must not have channels in its spec to avoid hitting PubNub)
+    mock_client = mock.Mock(spec=['scripts'])
+    mock_client.scripts = mock.Mock(spec=['get_sql_runs'])
+    response = MockResponse(state='success', output=expected)
```

Comment:
You should be able to use a real `Response` here and avoid creating a special object for testing.